pallets/avn-proxy/src/tests/mock.rs

```diff
@@ -113,6 +113,7 @@ impl pallet_nft_manager::Config for TestRuntime {
     type Public = AccountId;
     type Signature = Signature;
     type WeightInfo = ();
+    type BatchBound = ConstU32<10>;
```

Comment:
I think you mean to use `BatchNftBound` here?

Reply:
We can use any bound we want for testing purposes since this is configurable.